src/v2/providers/https.ts

```diff
@@ -356,11 +357,11 @@ export function onCall<T = any, Return = any | Promise<any>>(
  * @returns A function that you can export and deploy.
  */
 export function onCall<T = any, Return = any | Promise<any>>(
-  handler: (request: CallableRequest<T>) => Return
+  handler: (request: CallableRequest<T>, response?: CallableProxyResponse) => Return
 ): CallableFunction<T, Return extends Promise<unknown> ? Return : Promise<Return>>;
 export function onCall<T = any, Return = any | Promise<any>>(
   optsOrHandler: CallableOptions | ((request: CallableRequest<T>) => Return),
-  handler?: (request: CallableRequest<T>) => Return
+  handler?: (request: CallableRequest<T>, response?: CallableProxyResponse) => Return
 ): CallableFunction<T, Return extends Promise<unknown> ? Return : Promise<Return>> {
```

Comment:
TIL you can use the ternary operator with generics

Reply:
It's such weird TS feature

https://www.typescriptlang.org/docs/handbook/2/conditional-types.html